docs/victoriametrics/changelog/CHANGELOG.md

```diff
@@ -26,6 +26,7 @@ See also [LTS releases](https://docs.victoriametrics.com/victoriametrics/lts-rel
 
 ## tip
 
+* FEATURE: [vmalert](https://docs.victoriametrics.com/victoriametrics/vmalert/): add `now` function which return the Unix timestamp in second at the time of the template evaluation in VMRule alert for calculating the time difference in templates
```

Comment:
```suggestion
* FEATURE: [vmalert](https://docs.victoriametrics.com/victoriametrics/vmalert/): add `now` function that returns the current Unix timestamp in seconds. It can be used in templates, for example, to calculate duration the alert has been active `{{ (now | toTime).Sub $activeAt }}`. See this issue [#9864](https://github.com/VictoriaMetrics/VictoriaMetrics/issues/9864) for more details.
```